Assignments/Week2/non_repeating_character.py

```diff
@@ -0,0 +1,15 @@
+def non_repeating_character(s):
+    for i in range(len(s)):
+        if s[i] not in(s[:i]+s[(i+1):]):
```

Comment:
Not the most optimized way. Complexity is O(n^2).

Reply:
This is giving a Time Limit Exceeded Error on Geeksforgeeks. Don't submit PRs without having an accepted solution.

Reply:
No Dii, this is not giving any error on GFG. I have checked it again